apf/apfMesh.cc

```diff
@@ -242,18 +253,15 @@ bool Mesh::isParamPointInsideModel(ModelEntity* g,
     gmi_free_set(adjRegions);
     return true;
   }
-  // for faces with more than 1 adj model region return true for now
-  // TODO: update for future
-  if (adjRegions->n == 2) {
-    gmi_free_set(adjRegions);
-    return true;
-  }
+  (void)param;
+  (void)x;
+  return true; /*
   PCU_ALWAYS_ASSERT(adjRegions->n <= 1);
   gmi_ent* r = (gmi_ent*)adjRegions->e[0];
   gmi_eval(getModel(), (gmi_ent*)g, &param[0], &x[0]);
   int res = gmi_is_point_in_region(getModel(), r, &x[0]);
   gmi_free_set(adjRegions);
-  return (res == 1) ? true : false;
+  return (res == 1) ? true : false; */
```

Comment:
It looks like you break this function? Do we not test this?

Reply:
These changes were based on the discussion in #293 and I haven't looked at them since. It looks like @mortezah may have implemented something that at least partially addresses the issue, so my changes could probably be reverted.

Reply:
I'm not familiar with the exact functionality, so @mortezah might be able to shed more light. But, based on the discussion the solution of `return true` looks like a work around which needs to get fully fixed before merging?

Reply:
@tuckerbabcock the quick fix of returning true you put in place at the time we were discussing issue #293 is obsolete now. The code has changed since then and the current implementation of this function should work for you case, as well. If it did not, let me know.

@jacobmerson

Reply:
Thanks @mortezah. I'll revert these changes.